apps/L1G/tinder/tinder-frontend/src/components/GenderSelect.tsx

```
@@ -0,0 +1,58 @@
+'use client';
+
+import Image from 'next/image';
```

ESLint

[error] 'Image' is defined but never used.

(no-unused-vars)